components/arm/universalrobots/ur.go

```diff
@@ -277,11 +317,14 @@ func (ua *URArm) MoveToJointPositions(ctx context.Context, joints *pb.JointPosit
 
 // Stop stops the arm with some deceleration.
 func (ua *URArm) Stop(ctx context.Context, extra map[string]interface{}) error {
+	if !ua.inRemoteMode {
+		return errors.New("ur5 is in local mode switch to remote to control it")
```

Comment:
[minpr] can make this error a constant for readability since it shows up so much.